package.nls.zh-cn.json

```
@@ -1,75 +1,89 @@
-    "bookmarks.viewsWelcome.bookmarksExplorer.contents.whenNoFileOpen": "目前无书签。\n为了使用书签，你需要在编辑器中打开一个文件。\n[打开文件](command:workbench.action.quickOpen)\n可以[阅读文档](http://github.com/alefragnani/vscode-bookmarks/#bookmarks)了解关于 VS Code 中使用书签的更多信息。",
-    "bookmarks.viewsWelcome.bookmarksExplorer.contents.whenHasFileOpen": "目前无书签。\n为了使用书签，你需要将光标放在文件的任意位置，然后运行命令：\n[书签: 开关](command:bookmarks.toggle)\n可以[阅读文档](http://github.com/alefragnani/vscode-bookmarks/#bookmarks)了解关于 VS Code 中使用书签的更多信息。",
+    "bookmarks.viewsWelcome.bookmarksExplorer.contents.whenEmpty": "为了使用书签功能，你需要先打开一个文件夹或工作区。\n[打开文件夹](command:_bookmarks.openFolderWelcome)\n[打开工作区](command:workbench.action.openWorkspace)\n可以通过[阅读文档](http://github.com/alefragnani/vscode-bookmarks/#bookmarks)了解关于 VS Code 中使用书签的更多信息。",
+    "bookmarks.viewsWelcome.bookmarksExplorer.contents.whenNoFileOpen": "未找到任何书签。\n若需要编辑书签，你需要先在编辑器中打开一个文件。\n[打开文件](command:workbench.action.quickOpen)\n可以通过[阅读文档](http://github.com/alefragnani/vscode-bookmarks/#bookmarks)了解关于 VS Code 中使用书签的更多信息。",
+    "bookmarks.viewsWelcome.bookmarksExplorer.contents.whenHasFileOpen": "未找到任何书签。\n若需要添加书签，你需要先将光标放在文件的任意位置，然后运行命令：\n[书签: 添加 / 删除书签](command:bookmarks.toggle)\n可以通过[阅读文档](http://github.com/alefragnani/vscode-bookmarks/#bookmarks)了解关于 VS Code 中使用书签的更多信息。",
     "bookmarks.editor.context.label": "书签",
     "bookmarks.editor.title.label": "书签",
     "bookmarks.commands.category.bookmarks": "书签",
-    "bookmarks.commands.category.bookmarks.selection": "书签（选择）",
-    "bookmarks.commands.toggle.title": "开关",
-    "bookmarks.commands.jumpToNext.title": "跳至下一个",
-    "bookmarks.commands.jumpToPrevious.title": "跳至上一个",
-    "bookmarks.commands.jumpTo.title": "跳转至文档/行",
-    "bookmarks.commands.selectLines.title": "选中的行",
-    "bookmarks.commands.expandSelectionToNext.title": "将选择展开到下一个",
-    "bookmarks.commands.expandSelectionToPrevious.title": "将选择展开到上一个",
-    "bookmarks.commands.shrinkSelection.title": "收缩选择",
-    "bookmarks.commands.list.title": "列表",
-    "bookmarks.commands.toggleLabeled.title": "含标签的开关",
+    "bookmarks.commands.category.bookmarks.selection": "书签（选区操作）",
+    "bookmarks.commands.toggle.title": "添加 / 删除书签",
+    "bookmarks.commands.jumpToNext.title": "跳转至下一个书签",
+    "bookmarks.commands.jumpToPrevious.title": "跳转至上一个书签",
+    "bookmarks.commands.jumpTo.title": "跳转至文档 / 行",
+    "bookmarks.commands.selectLines.title": "选取所有书签行",
+    "bookmarks.commands.expandSelectionToNext.title": "扩展光标选区至下一个书签处",
+    "bookmarks.commands.expandSelectionToPrevious.title": "扩展光标选区至上一个书签处",
+    "bookmarks.commands.shrinkSelection.title": "收缩光标选区尾至选区中最后一个书签",
+    "bookmarks.commands.list.title": "列出文件中所有书签",
+    "bookmarks.commands.toggleLabeled.title": "添加含标签的书签 / 删除书签",
```

In Chinese writing habits, spaces are not usually added before and after punctuation marks. Therefore, a more reasonable sentence is:

```suggestion
    "bookmarks.commands.toggleLabeled.title": "添加含标签的书签/删除书签",
```